cmd/commands.go

```diff
@@ -291,6 +291,15 @@ Downloads an updated Caddy binary with the same modules/plugins at the
 latest versions. EXPERIMENTAL: May be changed or removed.`,
 	})
 
+	RegisterCommand(Command{
+		Name:  "add-package",
+		Func:  cmdAddPackage,
+		Usage: "<packageName>",
+		Short: "Adds Caddy package (EXPERIMENTAL)",
+		Long: `
+	  Adds caddy package`,
```

Comment:
Could we elaborate here a bit? Explain what happens, that existing non-standard packages will be retained, etc.

Reply:
Can you help me please write it well, if current version is not good enough🙏